### ISSUE TITLE
Why is relation id included in features ? 

### ISSUE THREAD

**Issue by Hadjerkhd** [FrankWork/conv_relation#13]
Hello, 
I couldn't understand why is rid included when building the `tf.train.SequenceExample()`, in the following lines : 
`rid = raw_example.label`
`ex.context.feature['rid'].int64_list.value.append(rid)`
Does it mean the rid is considered as a feature to train the cnn on ? 
Thanks in advance
